lib/bolt/inventory.rb

```diff
@@ -88,16 +92,37 @@ def groups_in(node_name)
       @groups.data_for(node_name)['groups'] || {}
     end
 
+    def transform_data(data)
+      if data
+        data['config'] = Bolt::Util.symbolize_keys(data['config']) if data['config']
+      end
+      data
+    end
+    private :transform_data
+
     # Pass a target to get_targets for a public version of this
     # Should this reconfigure configured targets?
     def update_target(target)
-      inv_conf = config_for(target.host)
-      unless inv_conf
-        @logger.debug("Did not find #{target.host} in inventory")
-        inv_conf = {}
+      data = @groups.data_for(target.host)
+      processed = transform_data(data) || {}
```

Comment:
I don't think we want to symbolize keys in the vars. That makes them hard to interact with from puppet.

Reply:
We don't symbolize the keys for vars, but this in place of calling 'config_for' which was originally symbolizing the keys for `data['config']`